src/utils/digitalAssets/compareDigitalAssetsByBalance.js

```diff
@@ -1,18 +1,27 @@
 // @flow
 
-import toBigNumber from 'utils/numbers/toBigNumber'
+import { divDecimals, toBigNumber } from 'utils/numbers'
+
+const toNominalValue = (asset: DigitalAssetWithBalance) => {
+  const { balance } = asset
+  const { decimals } = asset.blockchainParams
+  if (!balance || !decimals) {
+    return toBigNumber()
+  }
+
+  return divDecimals(
+    toBigNumber(balance.value),
+    decimals
+  )
+}
```

Comment:
```
return divDecimals(
    toBigNumber(balance.value),
    decimals
  )
```
could be just `return divDecimals(balance.value, decimals)`

Reply:
Thnx, fixed